src/Moryx.AbstractionLayer.Products.Endpoints/ProductManagementController.cs

```diff
@@ -163,10 +163,17 @@ public ActionResult<ProductModel> GetTypeById(long id)
         {
             if (id == 0)
                 return BadRequest($"Id was 0");
-            var productType = _productManagement.LoadType(id);
-            if (productType == null)
+            try
```

Comment:
Please simplify this 😬
var productType = null;
try
productType = _productManagement.LoadType(id);
catch 
...